### DISCUSSION TITLE
KEYWORD SEARCH VOLUME

### DISCUSSION THREAD

**Discussion by SinghAdit007** [etsy/open-api#1058] [Q&A]
Actually, we are trying to figure out the keyword search volume API, but not able to find one, can u please share the API from where I can fetch the keyword search volume.

**Comment by Gareth064** [comment]
There isn't one.

All the API's are documented. So if you can find something, that means it doesn't exist I'm afraid.

https://developer.etsy.com/documentation/reference

**Comment by SinghAdit007** [comment]
May be  i m not able to find one , can you please share one here, for
keyword search volume

**Reply by Gareth064** [reply]
Hey @SinghAdit007 , there isn't an API for what you are looking for.

**Reply by SinghAdit007** [reply]
is there any thing through we can get the keyword search volume indirectly, like incase u said direct API is not there, is there any indirect way to fetch it

**Reply by Gareth064** [reply]
There is not, I'm afraid. The closest thing to get "stats" is the view count on a listing.

**Reply by theapiguy** [reply]
I was looking for the same type of API.  EverBee and Alura have the information so I figured there was an API that returned that information.

**Reply by Gareth064** [reply]
I don't understand how those sites can offer that service. Unless they are doing some crazy level of scraping from Etsy's pages. Or some how, they have a deal with Etsy to get access to extra API's. Who knows? Etsy would never tell us if they did.

**Comment by Charlotte-br560** [comment]
Sure, you can try using the Google Ads Keyword Planner API or any other popular tool for keyword search volume data  .

**Comment by showtroylove** [comment]
They're using Google keyword data. It's not entirely related to searches intrinsic to Etsy; however, those are search engine keywords that resulted in traffic (extrinsically) to Etsy. It's a clever work-around but pales in comparison to the millions of foot traffic Etsy generates on its site.
